Automattic/jetpack
# Fix two year plans not showing up in my plan

There is an issue in the plugin that is preventing the new 2 year plans from showing up on the My Plan page, this PR fixes that

## Proposed changes:

* Add two year plan constants for all the products with new two year plans
* Also add Jetpack Stats yearly because it was missing
* Fixes a weird issue where Jetpack Scan Daily had the _Daily_ italicized for some reason

### Other information:


## Jetpack product discussion

P2: pbNhbs-81Y-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:
1. Get your local environment running with this branch or checkout this branch using the Jetpack Beta plugin on an ephemeral site
2. On your site, purchase a two year plan for all of the following products
 - Jetpack Social Advanced
 - Jetpack AI
 - Jetpack Stats
 - Jetpack Search
 - VaultPress Backup
 - Jetpack VideoPress
 - Akismet Anti-spam
 - Jetpack Scan
 - Jetpack Boost
3. Go to `/wp-admin/admin.php?page=jetpack#/my-plan` and make sure all of the products show up under "My Products"
![image](https://github.com/Automattic/jetpack/assets/65001528/347ae314-e90a-4c88-b8fb-765a66194cfe)
4. Next, purchase a 2 year Jetpack Security plan, and make sure it shows up under "My Plan"
![image](https://github.com/Automattic/jetpack/assets/65001528/bf842dbc-5322-425b-9265-6312cc332173)
5. Do the same with a Jetpack Complete 2 year plan
![image](https://github.com/Automattic/jetpack/assets/65001528/0599b352-5a5a-4ce6-80a0-870eea72ca8b)


NOTE: I found an issue when upgrading from JETPACK_SECURITY_TI_BI_YEARLY to JETPACK_COMPLETE_BI_YEARLY that needed to be fixed in WPCOM. It is fixed here if you'd like to test with it: D120873-code
